api/app/files/repository.py

```diff
@@ -0,0 +1,14 @@
+from app import db
+from app.files.models import File 
+
+class FileRepository:
+	@staticmethod
```

Comment:
Looks like some indentation is causing errors. Please correct.

Reply:
Hi @amritpurshotam. Get a bit confused with migration issues. I tried to do as you suggested by deleting the current migration file (Right ?) but it still doesn't work.

Reply:
Sorry forgot to mention you should delete your current file table on your local instance of your db. Use [pgAdmin](https://www.pgadmin.org/download/). Connect to `localhost` and use `docker` as both the username and password. Then you'll have a gui to delete the table.

Reply:
Hi @amritpurshotam. I've installed pgAdmin but I don't see exactly how to connect to my baoba local instance db. Sould I run docker also the same time ?

Reply:
@BrainTheos Yes `docker-compose up` will spin up the database instance and you should be able to connect

Reply:
@BrainTheos Something I also do to reset my database environment is to run the following query

```
DROP SCHEMA public CASCADE;
CREATE SCHEMA public;
```
Then run the upgrade command to rerun all the existing migrations. Then you can be sure your environment is up to date and synced correctly.

Reply:
@amritpurshotam , please look at this error i get when i run docker-compose up after lauching pgAdmin 
ERROR: for db  Cannot start service db: driver failed programming external connectivity on endpoint baobab_db_1 (387b839a1bba7b7a1dcace2243594b756c3206e1f2c85c2fc79755e0a15cb628): Error starting userland proxy: listen tcp 0.0.0.0:5432: bind: address already in use
ERROR: Encountered errors while bringing up the project.

Reply:
@BrainTheos this means you have another instance of the db container running. 
Try:
-  `docker-compose stop` or
 - `docker container ls` and then `docker stop [db_container_id]`

Reply:
@KaleabTessera thanks. it looks working. but still unable to see local db instance in pgAdmin (in Servers top left panel). and here is an other error i get though i tried to update my branch : 

Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade caniuse-lite browserslist`
webapp_1   | Failed to compile.
webapp_1   | 
webapp_1   | ./src/App.js
webapp_1   | Module not found: Can't resolve 'react-cookie-consent' in '/webapp/src'

Reply:
I think you may need to rebuild your docker image with `docker-compose build`

I remember I had that a little while back when the PR to add the cookie consent feature was merged in

Reply:
@BrainTheos Did yo manage to resolve the issue by rebuilding your image?

Reply:
@amritpurshotam , I didn't manage yet though i've rebuild my image. The problem is that my pgAdmin seems to not detect any database.

Reply:
@BrainTheos Let's discuss on slack on how to get you setup. For now to get things moving along, I've done the migration. This issue is now complete

Reply:
Ok @amritpurshotam, Thank you so much for your help and patience. I've really learned a lot from you !